documentation/modules/exploit/linux/http/ipcam_rce_with_creds_leak.md

```diff
@@ -0,0 +1,59 @@
+## Vulnerable Application
+  This module exploits an authenticated remote code execution in various branded ip cameras based on the same chinese OEM model.
+  Those cameras are also vulnerable to a badly implemented check on system.ini file within the embedded GoAhead web server.
+  By combining the two vulnerabilities, it is possible to achieve code execution without prior knowledge of any credentials.
+  1250+ Cameras models have been found vulnerable so far.
```

Comment:
Add a link to the way back machine's list of the models so people have a reference to look at in here